doc/src/manual/modules.md

```diff
@@ -144,8 +144,8 @@ In contrast,
 julia> import .NiceStuff
 ```
 
-brings *only* the module name into scope. Users would need to use `NiceStuff.DOG`, `NiceStuff.Dog`, and `NiceStuff.nice` to access its contents. Usually, `import ModuleName` is used in contexts when the user wants to keep the namespace clean.
-As we will see in the next section `import .NiceStuff` is equivalent to `using .NiceStuff: NiceStuff`.
+brings *only* the module name into scope. Users would need to use `NiceStuff.DOG`, `NiceStuff.Dog`, and `NiceStuff.nice` to access its contents.
+As we will see in the next section `import .NiceStuff` is equivalent to `using .NiceStuff: NiceStuff`. Usually, `import ModuleName` or `using ModuleName: ModuleName` is used in contexts when the user wants to keep the namespace clean.
```

Comment:
Why the sudden switch from `NiceStuff` to `ModuleName` ? I am fine with either name, but shouldn't it be used consistently, i.e. either everything with `NiceStuff` or else everything with `ModuleName`?

Reply:
That wasn't really my choice, I was just modifying the existing docs with its existing name scheme. See: https://docs.julialang.org/en/v1/manual/modules/#Standalone-using-and-import

FWIW I think maybe the that *could* be overhauled, but I also don't think it's necessary.